tuplex/test/wrappers/WrapperTest.cc

```diff
@@ -1552,6 +1552,25 @@ TEST_F(WrapperTest, TupleParallelizeI) {
     }
 }
 
+TEST_F(WrapperTest, Sort) {
+    using namespace tuplex;
+
+    PyObject* listObj = python::runAndGet("L = [5, 3, 1, 4, 2]", "L");
```

Comment:
can we expand this test a bit? I.e. sorting after different columns etc.? I'd also like to have one end-to-end test which sorts all the csv files after each column, and let's say combinations of 2-3 columns. Just to be sure the sorting feature works 100%.

Reply:
a good idea might be also, to vary the partitionSize for a large file, to be sure overlapping partitions are correctly handled.